### PR TITLE
Adds correct date variable in events/view/sessions

### LINKED ISSUE #307: Incorrect date variable in events/<id>/sessions
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current**
Variable used to pass date variable is incorrect in events/<id>/sessions. `speaker.submittedAt` is passed which does not exist in the model.

### PR DESCRIPTION
**Pull request by geekyd**
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adds correct date variable in events/view/sessions. Replaces `speaker.submittedAt` with `session.submissionDate` from the model function of route.

Fixes #307 
@fossasia/open-event-frontend Please review. Thanks

### CODE REVIEW

File: app/templates/events/view/sessions/list.hbs [hunk 1]
```diff
@@ -41,10 +41,10 @@
             {{session.shortAbstract}}
           </td>
           <td>
-            {{moment-format speaker.submittedAt 'dddd, DD MMMM YYYY'}}
+            {{moment-format session.submissionDate 'dddd, DD MMMM YYYY'}}
```
Comment (niranjan94) [inline app/templates/events/view/sessions/list.hbs]: would be called as `submittedAt`

File: app/templates/events/view/sessions/list.hbs [hunk 2]
```diff
@@ -41,10 +41,10 @@
...
           </td>
           <td>
-            {{moment-format speaker.submittedAt 'dddd, DD MMMM YYYY'}}
+            {{moment-format session.lastModified 'dddd, DD MMMM YYYY'}}
```
Comment (niranjan94) [inline app/templates/events/view/sessions/list.hbs]: would be called as modifiedAt

**Comment by geekyd** [comment]
@niranjan94 @CosmicCoder96 @sumedh123 Updated & made the requested changes. Please review. :)

**Comment by geekyd** [comment]
@niranjan94 @CosmicCoder96 Please merge this. Thanks :)

**Pull request merged**